src/Lykke.Service.Stellar.Api.Services/Transaction/TransactionService.cs

```diff
@@ -80,79 +84,90 @@ public async Task<TxBroadcast> GetTxBroadcastAsync(Guid operationId)
             return await _broadcastRepository.GetAsync(operationId);
         }
 
-        public async Task BroadcastTxAsync(Guid operationId, string xdrBase64)
+        public async Task BroadcastTxAsync(Guid operationId, string xdrBase64, TxBroadcast broadcast = null)
         {
             long amount = 0;
+            var xdr = Convert.FromBase64String(xdrBase64);
+            var reader = new ByteReader(xdr);
+            var txEnvelope = TransactionEnvelope.Decode(reader);
 
-            try
+            _chaos.Meow(nameof(BroadcastTxAsync));
```

Comment:
This call is unnecessary here, since there is no state changed before this call.

Reply:
Fixed